requirements/base.txt

```diff
@@ -34,6 +34,8 @@ pycares==4.3.0
     # via aiodns
 pycparser==2.21
     # via cffi
+typing-extensions==4.12.2 ; python_version < "3.11"
```

Comment:
There are other lockfiles that depend on base directly and indirectly. This needs to be reflected there as well.

Reply:
Are we fine with perhaps adding `typing-extensions` as a requirement regardless of version? There are also some 3.13 features we could start to use.

Mainly since I am seeing a lot of weirdness with the dependency resolution. Let's take `requirements/test.txt`. Running `pip-compile --allow-unsafe --output-file=requirements/test.txt --resolver=backtracking --strip-extras requirements/test.in` takes the existing `typing-extensions==4.10.0`, which is required by mypy, pydantic, etc, and adds < 3.11, which is obviously not what we want. Even explicitly adding it to `test.in` doesn't fix this.

Reply:
Fixed. https://github.com/aio-libs/aiohttp/pull/8463/commits/a5a61a7a6d0c73e16054e551e434efe889f55afc

Reply:
Whenever a `.in` file is updated, a cascade of corresponding `.txt` files update should follow. I see you dropped the conditional, but it's not reflected here. Though, if we don't actually use it above Python 3.11, I'd keep the conditional to reflect our actual direct use. It shouldn't affect the transitive dependencies if something else uses it.